doc/input-output-reference/src/overview/group-condenser-equipment.tex

```diff
@@ -3956,15 +3956,17 @@ \subsubsection{Inputs}\label{inputs-15-002}
 \item
   \textbf{CrossFlowSupplyMixedDemandUnMixed}.~ Specifes a single-pass, cross-flow heat exchanger.~ The effectiveness will be calculated using a cross-flow heat exchanger correlation for flow mixed on the Loop Supply side and flow unmixed on the Loop Demand Side.
 \item
-  \textbf{CrossFlowSupplyUnMixedDemandMixed}.~~ Specifes a single-pass, cross-flow heat exchanger.~ The effectiveness will be calculated using a cross-flow heat exchanger correlation for flow unmixed on the Loop Supply side and flow mixed on the Loop Demand Side.
+  \textbf{CrossFlowSupplyUnMixedDemandMixed}.~ Specifes a single-pass, cross-flow heat exchanger.~ The effectiveness will be calculated using a cross-flow heat exchanger correlation for flow unmixed on the Loop Supply side and flow mixed on the Loop Demand Side.
```

Comment:
@dareumnam can convert this to a description environment?

```
\begin{description}
  \item [CrossFlowWhatever] Description goes here.
  ...
\end{description}
```

This way you don't need to put periods at the end of each term name, no need for tilde ~ marks for spacing, etc.  Also, leave a space after the word and before you start the parentheses (like this).

Reply:
@Myoldmopar Thank you for the review. I fixed the code as your suggestion. I am going to pull a request now.